include/revng/Clift/Clift.h

```diff
@@ -23,6 +23,9 @@
 
 namespace mlir::clift::impl {
 
+inline constexpr unsigned BreakLabelFlag = 1;
+inline constexpr unsigned ContinueLabelFlag = 2;
```

Comment:
If we're going to use this as a flag, which is what we do, let's use `1 << 1` instead of `2`.